test/rulesets/LinearAlgebra/norm.jl

```diff
@@ -1,3 +1,5 @@
+@eval ChainRulesTestUtils check_thunking_is_appropriate(_) = nothing
```

Comment:
This needs to be removed before we can merge this.
```suggestion
```

Reply:
Yes. But see https://github.com/JuliaDiff/ChainRules.jl/pull/337#discussion_r623430417 above, why is this tested at all?